dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -136,6 +136,12 @@
      */
     private int hangBufferPoolSize = 0;
 
+    /**
```

Comment:
Copyright should be updated to say 2012, 2020 instead of 2012, 2016

Reply:
Done